crates/cli/src/commands/setup.rs

```diff
@@ -132,10 +113,11 @@ pub async fn setup(args: ArgsRef<SetupArgs>, proto: ResourceRef<ProtoResource>)
 
     // If we found a profile, update the global config so we can reference it
     if let Some(profile) = &profile_path {
+        println!("Updated profile {}", color::path(profile));
```

Comment:
This would appear before the success messages right? Feels a bit weird.

Reply:
Yup. 

I dunno, not weird to me at least. Isn't that usually what CLIs do, log progress as they go?

Reply:
Just gonna trust your gut on this. I've moved it back, although its still on its own line after the success message, due to all the message variations now. 

The finished_message has gotten a bit messy, if you have an idea for restructuring the function I'm all ears :)